plugins/parcelport/verbs/scheduler.hpp

```diff
@@ -0,0 +1,132 @@
+//  Copyright (c) 2015-2016 John Biddiscombe
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
```

Comment:
Is this file still needed? Wasn't that superseded by one of the executor classes we added?